web-test-runner.config.mjs

```diff
@@ -37,16 +37,30 @@ export default {
   ],
   concurrentBrowsers: 3,
   browsers: [
-    playwrightLauncher({ product: 'chromium' }, {
-      headless: true,
-      args: [
-        '--disable-setuid-sandbox',
-        '--disable-extensions'
-      ]
+    playwrightLauncher({
+      product: 'chromium',
+      createBrowserContext: ({browser}) => browser.newContext({ignoreHTTPSErrors: true}),
+      launchOptions: {
+        headless: true,
```

Comment:
I am not sure that do we really need to set `headless` to true ? . I just went to the type description of `headless` property and they said that `headless` was set to `true` by default.

Reply:
Yes, we can remove it for clean code. 😀

Reply:
removed